spec/models/user_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
Remove if not in use

Reply:
Or add tests. I don’t understand why there are no unit tests for User class